src/models/bets.ts

```diff
@@ -27,6 +27,9 @@ const betsSchema: Schema = new mongoose.Schema(
     description: {
       type: String
     },
+    isfree: {
```

Comment:
As a developer we need to enhance our readability using camelCase, PascalCase or snake_case

in our case, due to the conventions we have worked on the entire project we should use one of them instead of writing words in this way, please do the change properly and correct the impact of it

Reply:
Understood